.travis.yml

```diff
@@ -1,21 +1,29 @@
----
 language: node_js
 node_js:
-  - "6"
-
+- '6'
 sudo: false
 dist: trusty
-
 addons:
   chrome: stable
-
+  sonarcloud:
+    organization: emberfaces
+    token:
+      secure: 
+    branches:
+    - master
+    - develop
```

Comment:
This is wrong. It should not be under addons/sonarcloud, but in the root. Plus it's not complete, look 
https://github.com/voxpupuli/modulesync_config/commit/bcccf0035903b35006d0f65af3ac708eaf20f8d6